test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointSmokeTest.java

```diff
@@ -81,30 +81,30 @@ public void smoke1Test() throws Exception {
         final String checkpointAuthor = "Hey, it's me!";
 
         // Put keys 1 & 2 into m
-        Map<String, Long> m = instantiateMap(streamName);
-        m.put(key1, key1Val);
-        m.put(key2, key2Val);
+        Map<String, String> m = instantiateMap(streamName);
```

Comment:
not sure why are we changing the type to String?

anyway, I think these tests need some style cleanup, let'e defer to a different PR, and not stall on this one, because we want to get the performance improvements asap.